Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -239,6 +239,14 @@ open class ChatChannelVC:
             channelController.markRead()
         }
         messageListVC.updateMessages(with: changes)
+        
+        guard let cooldownDuration = channelController.channel?.cooldownDuration,
+              cooldownDuration != messageComposerVC.content.countdownDuration else {
+            return
+        }
+
+        messageComposerVC.content.countdownDuration = cooldownDuration
+        messageComposerVC.checkChannelCountdown()
```

Comment:
Ideally, calling `messageComposerVC.updateContent()` would be enough here, if we move the logic to the updateContent() only.

Reply:
Done